test-case/verify-sof-firmware-load.sh

```diff
@@ -35,7 +35,8 @@ if $cmd | grep -q " sof-audio.*Firmware.*version"; then
     exit 0
 else
     journalctl --dmesg --lines 50
-    journalctl --dmesg | grep -C 1 -i version
-    printf '$cmd was %s\n' "$cmd"
+    journalctl --dmesg | grep -q " sof-audio.*Firmware.*version";
+    printf '$cmd was %s, DMESG_LOG_START_LINE was %s\n' "$cmd" "$DMESG_LOG_START_LINE"
+    $cmd
```

Comment:
The output of `$cmd` could be huge in case of a bug so how about:
- running it again first, immediately after the `else` so the other, shorter output don't get lost
- add a safety like: `$cmd | tail -n 10000`
- add some separators `---`.

```shell
else
     $cmd
    printf ' ------\n $cmd was %s, DMESG_LOG_START_LINE was %s  \n ---- \n' \
            "$cmd" "$DMESG_LOG_START_LINE"
    journalctl --dmesg --lines 50
    journalctl --dmesg | grep -C 1 " sof-audio.*Firmware.*version" || true
```